scalajslib/src/mill/scalajslib/ScalaJSModule.scala

```diff
@@ -50,6 +50,21 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
     )
   }
 
+  def scalaJSJsEnvIvyDeps: Target[Agg[Dep]] = T {
```

Comment:
I'm very happy with this change which will speed up downloading dependencies in Scala.js builds, since now Mill downloads only the dependency you need instead of downloading all of them :)